src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -296,55 +296,47 @@ public void ProcessStart_UseShellExecute_OnUnix_ValidVerbs(string verb, bool isV
             }, verb ?? "<null>", isValid.ToString(), options).Dispose();
         }
 
-        [Theory, InlineData("vi")]
-        [PlatformSpecific(TestPlatforms.Linux)]
-        [OuterLoop("Opens program")]
-        public void ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram(string programToOpenWith)
+        [Fact]
+        public void ProcessStart_OnLinux_UsesSpecifiedProgram()
         {
-            if (IsProgramInstalled(programToOpenWith))
+            const string Program = "sleep";
+
+            using (var px = Process.Start(Program, "60"))
             {
-                string fileToOpen = GetTestFilePath() + ".txt";
-                File.WriteAllText(fileToOpen, $"{nameof(ProcessStart_OpenFileOnLinux_UsesSpecifiedProgram)}");
-                using (var px = Process.Start(programToOpenWith, fileToOpen))
+                try
+                {
+                    Assert.Equal(Program, px.ProcessName);
+                }
+                finally
                 {
-                    Assert.Equal(programToOpenWith, px.ProcessName);
                     px.Kill();
                     px.WaitForExit();
                     Assert.True(px.HasExited);
```

Comment:
Nit: probably better to move this assert out of the finally.  We don't need this assert to run if the previous one fails, and if they both failed, we'd actually want the exception message from the first to propagate.